assets/javascripts/listSort.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable */
+
+$(document).ready(init)
```

Comment:
any point having the `init` function as part of the global scope? probably worth just sticking it in the ready as a callback?

Reply:
Nope, no point. Have brought it inline